gr-utils/python/modtool/gr-newmod/swig/howto_swig.i

```diff
@@ -2,7 +2,7 @@
 
 #define HOWTO_API
 
-%include "gnuradio.i"			// the common stuff
+%include "gnuradio.i"           // the common stuff
```

Comment:
Was this accidental?

Reply:
No, I have replaced `tab spaces` which look not so good using `vim`.